docs/docs/writing-your-own-enclave-host.md

```diff
@@ -277,20 +276,17 @@ public class MyEnclaveHost {
 }
 ```
 
-!!! tip
-
-    In projects containing multiple enclave modules, the enclave to load can be specified by passing the fully qualified class name like so:
+!!!Note
```

Comment:
Minor thing. I notice that here we have `!!!Note`, while elsewhere we have `!!! Note`, with a space in it. I think we should be consistent (not sure which one is rendered correctly when published, though)

Reply:
Both works. I have made it consistent without space in this doc. I am doing the same in other docs. I missed it here; thanks for spotting it.